src/interactive/scala/tools/nsc/interactive/tests/core/AskCommand.scala

```diff
@@ -121,7 +122,7 @@ trait AskTypeAt extends AskCommand {
 trait AskLoadedTyped extends AskCommand {
   import compiler.Tree
 
-  protected def askLoadedTyped(source: SourceFile, keepLoaded: Boolean = false)(implicit reporter: Reporter): Response[Tree] = {
+  protected def askLoadedTyped(source: SourceFile, keepLoaded: Boolean = false)(implicit @unused reporter: Reporter): Response[Tree] = {
```

Comment:
`implicit @unused` or `@unused implicit`? Why isn't this part of the style guide?

Reply:
💯

Reply:
It's part of the spec. `(implicit params...)` and `@annot id`. Less obvious with just one param.